Content/Items/Food/Meal.cs

```diff
@@ -34,6 +34,15 @@ public override void SetDefaults()
 			Item.height = 32;
 		}
 
+		public override ModItem Clone(Item newEntity)
+		{
+			var clone = base.Clone(newEntity) as Meal;
+			var newList = new List<Item>();
+			Ingredients.ForEach(n => newList.Add(n.Clone()));
```

Comment:
Is cloning the ingredients necessary? Yes if `OnUseEffects` may mutate them. 
If no, and the `Ingredients` list can never be changed after crafting a `Meal`, then you may not need to clone the list at all. Since a copy of a meal has the same ingredients.

Reply:
Not currently though its possible that in the future they may hold instanced data? This seems to be the safer course of action

Reply:
Judging by your `NetSend`, which only sends the `type`, I think you'd be better off storing a `List<int>` of the item ids, and using `ContentSamples.ItemsByType` (or another common list of food item instances)